test/unit/common/include/monad/test/monad_revision_test.hpp

```diff
@@ -122,6 +122,16 @@ struct MonadEvmRevisionTest : public ::testing::Test
         }
     }
 
+    static constexpr bool is_monad_revision()
+    {
+        return std::same_as<typename T::value_type, monad_revision>;
+    }
+
+    static constexpr bool is_evmc_revision()
```

Comment:
Tiny change (not consequential but matches how this kind of method is defined elsewhere):

```suggestion
    static consteval bool is_monad_revision() noexcept
    {
        return std::same_as<typename T::value_type, monad_revision>;
    }

    static constexpr bool is_evmc_revision() noexcept
```